Dockerfile

```diff
@@ -1,39 +1,28 @@
-FROM java:7
+FROM java:8-jdk
 MAINTAINER Gabe Conradi <gabe@tumblr.com>
 
 RUN apt-get update && apt-get install -y zip unzip && rm -r /var/lib/apt/lists/*
 
-RUN useradd -Ur -d /opt/collins collins
-RUN for dir in /build /build/collins /var/log/collins /var/run/collins; do mkdir $dir; chown collins $dir; done
+ENV ACTIVATOR_VERSION 1.3.7
 
-WORKDIR /build
 # get Play, Collins, build, and deploy it to /opt/collins
 COPY . /build/collins
-RUN wget -q http://downloads.typesafe.com/typesafe-activator/1.3.6/typesafe-activator-1.3.6-minimal.zip -O /build/typesafe-activator-1.3.6-minimal.zip && \
-    unzip -q ./typesafe-activator-1.3.6-minimal.zip && \
+RUN cd /build && \
+    wget -q http://downloads.typesafe.com/typesafe-activator/$ACTIVATOR_VERSION/typesafe-activator-$ACTIVATOR_VERSION-minimal.zip -O /build/activator.zip && \
+    unzip -q ./activator.zip && \
     cd collins && \
     java -version 2>&1 && \
-    PLAY_CMD=/build/activator-1.3.6-minimal/activator ./scripts/package.sh && \
+    PLAY_CMD=/build/activator-$ACTIVATOR_VERSION-minimal/activator ./scripts/package.sh && \
     unzip -q /build/collins/target/collins.zip -d /opt/ && \
-    cd / && rm -rf /build && \
-    chown -R collins /opt/collins
+    cd / && rm -rf /build
 
-# Add in all the default configs we want in this build so collins can run.
-# Override /opt/collins/conf with your own configs with -v
-COPY conf/docker/validations.conf     /opt/collins/conf/validations.conf
-COPY conf/docker/authentication.conf  /opt/collins/conf/authentication.conf
-COPY conf/docker/database.conf        /opt/collins/conf/database.conf
-COPY conf/docker/production.conf      /opt/collins/conf/production.conf
-COPY conf/docker/users.conf           /opt/collins/conf/users.conf
-COPY conf/docker/profiles.yaml        /opt/collins/conf/profiles.yaml
-COPY conf/docker/permissions.yaml     /opt/collins/conf/permissions.yaml
-COPY conf/docker/logger.xml           /opt/collins/conf/logger.xml
+WORKDIR /opt/collins
 
-RUN chown -R collins /opt/collins
+# Add in all the default configs we want in this build so collins can run.
+# You probably will want to override these configs in production
+COPY conf/docker conf/
```

Comment:
Agreed that a set of default configs should be baked into the image.

It would be nice to expose the user-serviceable configs by adding `VOLUME /opt/collins/conf` in the Dockerfile, as described at https://docs.docker.com/engine/articles/dockerfile_best-practices/#volume